packages/neon/neon_talk/lib/src/blocs/room.dart

```diff
@@ -26,7 +26,7 @@ abstract class TalkRoomBloc implements InteractiveBloc {
   BehaviorSubject<Result<spreed.Room>> get room;
 
   /// The messages in this room.
-  BehaviorSubject<Result<BuiltList<spreed.ChatMessageWithParent>>> get messages;
+  BehaviorSubject<Result<BuiltMap<int, spreed.ChatMessageWithParent>>> get messages;
```

Comment:
Messages need to be unique by their ID, but the default comparison of all properties will not work because some might change. Is there a more clever way to do this?